### ISSUE TITLE
NoMethodError: undefined method `get_app_portal_access_access_api_v1_auth_dashboard_access_app_id_post'

### ISSUE THREAD

**Issue by gottfrois** [svix/svix-webhooks#910]
## Bug Report

When trying to get the app portal access link using the Ruby client I get the following behavior:

```ruby
client = Svix::Client.new('AUTH_TOKEN')
client.authentication.app_portal_access('APPLICATION_ID', Svix::AppPortalAccessIn.new({}))
NoMethodError: undefined method `get_app_portal_access_access_api_v1_auth_dashboard_access_app_id_post' for #<Svix::AuthenticationApi:0x0000000130427c60>
Did you mean?  get_app_portal_access_api_v1_auth_app_portal_access_app_id_post
               get_dashboard_access_api_v1_auth_dashboard_access_app_id_post
from /Users/xxx/.asdf/installs/ruby/2.7.6/lib/ruby/gems/2.7.0/gems/svix-0.83.1/lib/svix/authentication_api.rb:10:in `app_portal_access'
```

Documented [here](https://docs.svix.com/app-portal#giving-your-users-access).

### Version

```
0.83.1
```

### Platform

```sh
$ uname -a
Darwin xxx 22.4.0 Darwin Kernel Version 22.4.0: Mon Mar  6 20:59:58 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T6020 arm64
```

```sh
$ ruby -v
ruby 2.7.6p219 (2022-04-12 revision c9c2245c0a) [arm64-darwin22]
```

**Comment by tasn** [comment]
Whoops, never saw the report! We'll take a look!

**Comment by svix-lucho** [comment]
@gottfrois just verified that this is fixed in the latest version of the gem (`1.8.1`). Please update to the latest version and try again 😃
